### PR TITLE
Update documentation and fix use of Select interaction :)

### PR DESCRIPTION
**Pull request by oterral**
- update documentation of parameters for some controls
 - Replace `source` property by `layers` property for Select interaction
 - Apply the select style on top of the feature's style.
- Prettify the html

### CODE REVIEW

File: src/control/delete.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ class DeleteControl extends Control {
      * @private
      */
     this.selectInteraction = new ol.interaction.Select({
-      source: this.source,
+      layerFilter: this.layerFilter,
```
[the remark this answers is not in the document]
Comment (oterral) [reply src/control/delete.js]: totally

File: src/control/control.js [hunk 2]
```diff
@@ -63,6 +61,16 @@ class Control extends ol.control.Control {
...
       });
 
+    /**
+     * Filter editable layer. Used by select interactions instead of
+     * the old source property.
+     * @type {function}
+     * @private
+     */
+    this.layerFilter =
+      options.layerFilter ||
```
[the remark this answers is not in the document]
Comment (oterral) [reply src/control/control.js]: it was like this for the other property.

File: index.html [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
-    <link rel="stylesheet" type="text/css" href="style/style.css">
-    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="style/style.css" />
+    <meta charset="utf-8" />
   </head>
   <body>
     <script src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/build/ol.js"></script>
     <script src="https://unpkg.com/jsts@2.0.3/dist/jsts.min.js"></script>
-    <script src="build/index.js"></script>
+    <script src="index.js"></script>
```
Comment (oterral) [inline index.html]:
```suggestion
    <script src="build/index.js"></script>
```

File: src/control/control.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (friedjoff) [reply src/control/control.js]: Ok, makes sense, but let's also document this property in the contructor options.

[the remark this answers is not in the document]
Comment (oterral) [reply src/control/control.js]: Right, I hadn't seen these definitions.

**Pull request merged**